### DISCUSSION TITLE
Code Removal

### DISCUSSION THREAD

**Discussion by symful** [community/community#121584] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Can I request GitHub copyright to remove my code from a repository from an organization that had removed me?

**Comment by ebndev** [comment]
Hi @ardofulian,

This community is managed by GitHub employees, but it is not a support forum. 

For [Submitting content removal requests](https://docs.github.com/en/site-policy/content-removal-policies/submitting-content-removal-requests), you can review the documentation and see if one of those policy procedures are relevant to you.

Best of luck!
